readme.usage.md

```diff
@@ -18,7 +18,28 @@ placing it into the topmost parent directory of your repo.
 
 Optionally, you can use another filename and location, and then specify it as a command line argument.
 
-#### 2- sesamconfig
+#### 2- authconfig
+
+In case of using a connector (e.g. Hubspot connector), Sesam client uses `.authconfig` file to authenticate via oauth or 
+Tripletex and update secrets and environmental variables. To utilize authconfig create `.authconfig` in the working 
```

Comment:
Absolute super nitpick - environment variables, not environmental :)

Reply:
done:)